.github/workflows/validate-shell-scripts-ubuntu.yml

```diff
@@ -23,10 +23,13 @@ on:  # yamllint disable-line rule:truthy
 jobs:
   lint:
     uses: ./.github/workflows/lint.yml
-  validate-shell-scripts:
+  validate-shell-scripts-ubuntu:
+    strategy:
+      matrix:
+        os: [ubuntu-22.04]
```

Comment:
Can you please use a single workflow file, instead of two by adding `macos-latest` to this set and removing the MacOS-specific workflow?

Reply:
updated